# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1323663607 18000
#      Sun Dec 11 23:20:07 2011 -0500
# Node ID 2ed17790425e98430574eac3319990338384f598
# Parent  79c8a524336031a7bb10accc7b8aab2c3c1ce344
Added tag ss-3-5-90 for changeset 3781981be535

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -48,8 +48,9 @@ 79a56d0a6a0d693ebf461c456951b20dfbcde140
 229675bb7647f7e7e499bcb5acacc8eabe7f3697 ss-3-3-52
 6c69a7c39039bccd5a2fe481fcc7f896c82b160b ss-3-3-53
 3cbc0d77db48aec32bcb202d09a036d2cb9cc3b9 ss-3-3-53
 bd2643f0ce57d070963bedd48857405f6924aa85 ss-3-3-54
 695141f1c05cf1b240592bdd18e7a1503bb2a539 ss-3-3-55
 901d466ee55ac902a875ec0ade6f1eccef0841dc release-3-4-1
 3666e8e6f96e6899b8306d6ea9614aadf0500d67 release-3-4-2
 b0e70a71647b671ebcfa7a79af1ae6d3c0f52065 release-3-4-3
+3781981be535e80d44c85373b8fdaa60ca5cd097 ss-3-5-90
